### ISSUE TITLE
  Able to add <?predicate?> as string

### ISSUE THREAD

**Issue by efotopoulou** [LinDA-tools/LindaWorkbench#197]
At step 4 , sometimes the vocabulary the end user wants to use are not retrievable, but still the end user may feel obligated to use specific vocabularies in order to achieve an interlinking he has in mind.

It would be really practical to be able to add a url as string in order to have the desirable result.

For example add http://purl.org/linked-data/sdmx/2009/dimension/refPeriod as simple string.

**Comment by lfarid** [comment]
yes, we have this on the list of todo's. see issue https://github.com/LinDA-tools/transformation/issues/9. this will allow the user to manually enter a predicate's URI (also for reconciling entities and for classes).
